app/index.js

```diff
@@ -150,12 +151,12 @@ Generator.prototype.ohTellMeWhatYouWantWhatYouReallyReallyWant = function() {
 Generator.prototype.justIgnoreMe = function() {
 	if (this.conf.get('git')) {
 		this.logger.verbose('Copying .gitignore file');
-		this.copy('gitignore.tmpl', '.gitignore');
-		this.logger.verbose('Done copying .gitignore file');
+		this.fs.copyTpl(this.templatePath('gitignore.tmpl'), this.destinationPath('.gitignore'), { conf: this.conf });
+		//this.logger.verbose('Done copying .gitignore file');
```

Comment:
Can you clean up the comments?  For these log ones we should just uncomment them.